Sync with v.1.0.0 release of metadatascheme (#11)
Health-RI/metadata-shacl-validation: README.md

This repository is used to provide SHACL shapes to verify Health-RI and FAIR Data Point RDF files.

> The validator itself is available at <https://www.itb.ec.europa.eu/SHACL/healthri/upload>.

Reviewer: The uppercase link seems to not redirect properly to the lowercase version. I'd suggest to revert back to the lowercase one.

```suggestion
The validator itself is available at <https://www.itb.ec.europa.eu/shacl/healthri/upload>.
```

Author: Yes you're right... i was a overactive when replacing 'shacl' by 'SHACL' as mark asked...

> The validator itself is available at <https://www.itb.ec.europa.eu/SHACL/healthri/upload>.

Reviewer 2: ```suggestion
The validator itself is available at <https://www.itb.ec.europa.eu/shacl/healthri/upload>.
```
all caps 404s

Author: fixed in last commit

## Development

Run `docker compose build && docker compose up` and a validator will be exposed at <http://localhost:8080/shacl/healthri/upload>.

> Currently there are two profiles v.1.0.0 and development. The v1.0.0. profile will validate against the v1.0.0 release shacls, 

Reviewer 2: ```suggestion
Currently there are two profiles: v.1.0.0 and development. The v1.0.0. profile will validate against the v1.0.0 release SHACLs, 
```

> development will validate against development, note: development can contains errors, so use with care!.

Reviewer 2: ```suggestion
development will validate against the latest development version. Note: development can contains errors, so use with care!
```

> The shacles are retrieved from github datamodel repository.

Reviewer 2: ```suggestion
The SHACLs are retrieved directly from the github datamodel repository.
```

## Updating existing shapes

Updating is not needed anymore, the shacl are retrieved from github datamodel repository.

## Adding new shapes

To add a new version/profile, edit *config.properties* file. At the first line, add a new profile to the
`validator.type` line. Then, add two lines: `validator.typeLabel.<your profile> = <profile description>`
this is the title that would appear in the drop down menu.

> `validator.shaclFile.<your profile>.remote.<#>.url = http://..... and 

Reviewer 2: ```suggestion
`validator.shaclFile.<your profile>.remote.<#>.url = http://.....` and
 
```

> `validator.shaclFile.<your profile>.remote.<#>.type = text/turtle

Reviewer 2: ```suggestion
`validator.shaclFile.<your profile>.remote.<#>.type = text/turtle`
```
